firefox-ios/Client/Frontend/Browser/TabWebViewPreview/Redux/TabWebViewPreviewAction.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Redux
+
+final class TabWebViewPreviewAction: Action {
+    let addressBarPosition: SearchBarPosition?
+
+    init(addressBarPosition: SearchBarPosition?, actionType: ActionType) {
+        self.addressBarPosition = addressBarPosition
+        super.init(windowUUID: .unavailable, actionType: actionType)
```

Comment:
why `.unavailable`? should we pass in the action.windowUUID instead?

Reply:
Feature should be available only available for iPhones.

Reply:
can we add a comment here to clarify that?